packages/icons/src/v4/SAP-icons.json

```diff
@@ -1854,7 +1857,10 @@
       "path": "M261.5 197q-6-6-11 0l-160 160q-5 5-11.5 5t-11.5-5-5-11.5 5-11.5l166-165q9-9 22-9t23 9l165 165q5 5 5 11t-5 11q-12 12-23 0z"
     },
     "soccer": {
-      "path": "M201.902 256.966l17.389 90.808-72.453 36.709-74.385-55.064 36.71-91.774zM347.774 72.453l55.064 55.064-37.676 74.385-90.807-19.32V91.773zM256.966 0q53.132 0 99.502 20.287t81.147 55.064 54.581 81.63T512 256.966t-19.804 99.502-54.58 81.147-81.148 54.581T256.966 512q-54.098 0-100.95-19.804t-81.631-54.58-54.581-81.148T0 256.966t19.804-99.985 54.58-81.63 81.631-55.064T256.966 0zm200.936 374.823q16.423-27.05 26.566-56.997t10.143-60.86l-31.879-111.094q-33.811-55.064-88.392-91.774t-117.374-36.71q-46.37 0-92.74 22.22v32.845l-72.452 74.385H49.268q-14.49 25.117-23.185 53.132t-8.694 56.996q0 29.947 9.177 58.928t25.117 55.064 37.675 48.302 47.82 38.642h9.66l110.128 36.71q33.811 0 66.174-12.076t61.343-31.396l-36.71-11.593-18.354-55.064 73.419-74.385 55.064 37.676v27.049z"
+      "path": "M256.5 1q53 0 99.5 20.5t81 55 54.5 81 20 100.5q0 53-20 99T437 437.5 356 492t-99.5 20q-54 0-100.5-20t-81.5-54.5T20 357 .5 258q0-54 19.5-100.5t54.5-81 81.5-55T256.5 1zm201 374q16-27 26-57t10-60l-31-111q-34-55-88.5-92T257.5 18q-47 0-93 23v32l-73 75h-42q-14 25-23 53t-9 57 9.5 58 25 55 37.5 48.5 48 38.5h9l110 37q34 0 66.5-12t61.5-32l-37-11-18-55 73-74 55 37v27zm-110-302l55 55-38 75-90-20V93zm-146 185l18 90-73 37-74-55 37-92z"
+    },
+    "soccor": {
+      "path": "M256.5 1q53 0 99.5 20.5t81 55 54.5 81 20 100.5q0 53-20 99T437 437.5 356 492t-99.5 20q-54 0-100.5-20t-81.5-54.5T20 357 .5 258q0-54 19.5-100.5t54.5-81 81.5-55T256.5 1zm201 374q16-27 26-57t10-60l-31-111q-34-55-88.5-92T257.5 18q-47 0-93 23v32l-73 75h-42q-14 25-23 53t-9 57 9.5 58 25 55 37.5 48.5 48 38.5h9l110 37q34 0 66.5-12t61.5-32l-37-11-18-55 73-74 55 37v27zm-110-302l55 55-38 75-90-20V93zm-146 185l18 90-73 37-74-55 37-92z"
```

Comment:
In openui5, there are  "soccer" and "soccor"(I think it's typo), but I guess we should also support both names for the case of UI5 Web Components Consumption in OpenUI5